app/parse.py

```diff
@@ -1,15 +1,119 @@
+import csv
 from dataclasses import dataclass
+import requests
+from bs4 import BeautifulSoup
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)